src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -34,10 +34,38 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String preppedSentence = sentence;
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
+        System.out.println(preppedSentence);
+        System.out.println(Arrays.toString(wordsInPreppedSentence));
         return Arrays.stream(wordsInPreppedSentence)
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case, but a full word match is required.
+     *   <br>examples:<pre>
+     *       containsWordIgnoreCase("ABc def", "abc") == true
+     *       containsWordIgnoreCase("ABc def", "DEF") == true
+     *       containsWordIgnoreCase("ABc def", "AB") == false //not a full word match
+     *       </pre>
+     * @param sentence cannot be null
+     * @param word cannot be null, cannot be empty, must be a single word
+     */
```

Comment:
I think you can update the javadoc comment

Reply:
Resolved. Thanks!